### DISCUSSION TITLE
Garmin Watch not detected - iOS Devices are

### DISCUSSION THREAD

**Discussion by Dtwo-sys** [mKeRix/room-assistant#412] [General]
Hi, new to all of this, have installed on the Rpi running Home Assistant and a second Pi W - have added two iphones, one apple watch and a gramin watch. HA has automatically picked up the iOS devices but not gramin watch - I have checked the MAC address; is there some further step I need to take?

Thanks

**Comment by mKeRix** [comment]
At least I don't know of any special steps that would be required for Garmin watches, but don't own one or have tried tracking one myself. Are you trying to track via BT Classic or BLE? For the Garmin you might have to use Classic.

Apart from that, recently there was someone in #382 that managed to track a Garmin device - maybe that user also has some more insights?

**Reply by Dtwo-sys** [reply]
thank you, appreciate your time to comment

**Reply by AdmiralRaccoon** [reply]
I bought a Garmin Forerunner yesterday, and also tried room-assistant with the BLE integration, and it works fine so far - as long I disable bluetooth on my phone, so the garmin app can´t connect to the watch.
When activating, room-assistant sets the watch almost instantly to "not_home"

Is this a general limitation to BLE that it doesn´t send advertisements while connected to one device?

**Reply by AdmiralRaccoon** [reply]
Sorry, found the answer in a previous post - seems a known issue within the smartwatches
